scripts/lib.sh

```diff
@@ -70,6 +70,14 @@ start-docker() {
         # At a later stage in the script, we're going to execute
         # ensure_docker_server which waits for it to be ready
         open -g -a Docker.app
+        exit_code=$?
+        # If the step above fails, at least we can get some debugging information to determine why
+        if [ $exit_code != 0 ]; then
```

Comment:
You can just replace all this with `if ! open -g -a Docker.app; then`.

Reply:
True true! Maybe one day bash will come natural